### ISSUE TITLE
CMSIS Pack Manager: would be nice to provide API for device selection

### ISSUE THREAD

**Issue by marek-trmac** [gnuarmeclipse/plug-ins#97]
It would be good to have device assigned in the newly created project (in project Properties - C/C++ Build - Settings - Devices). CMSIS Pack manager should provide API allowing to assign device, so new project wizards can leverage.

**Comment by ilg-ul** [comment]
yes, the future project wizard will do this, but it'll take a while for it to be ready.
